model/storage/metrics.yaml

```diff
@@ -0,0 +1,100 @@
+groups:
+  # storage.* metrics
+  - id: metric.storage.volume.disk.count
+    type: metric
+    metric_name: storage.volume.disk.count
+    stability: development
+    brief: "Number of active/failed/spare disks of device."
+    note: |
+      Reports the total number of disks in a RAID device.
```

Comment:
Should these descriptions still say `RAID` if the namespace is just `storage` now?